common/djangoapps/student/tests/test_activate_account.py

```diff
@@ -17,13 +20,31 @@ def setUp(self):
         super(TestActivateAccount, self).setUp()
         self.username = "jack"
         self.email = "jack@fake.edx.org"
-        self.user = User.objects.create(username=self.username, email=self.email, is_active=False)
+        self.password = "test-password"
+        self.user = UserFactory.create(
+            username=self.username, email=self.email, password=self.password, is_active=False,
+        )
 
         # Set Up Registration
         self.registration = Registration()
         self.registration.register(self.user)
         self.registration.save()
 
+    def login(self):
+        """
+        Login with test user.
```

Comment:
Please either add some comment about preserving the account activation status or update the helper method name.